src/client/theme-default/without-fonts.ts

```diff
@@ -14,10 +14,12 @@ import Layout from './Layout.vue'
 // Note: if we add more optional components here, i.e. components that are not
 // used in the theme by default unless the user imports them, make sure to update
 // the `lazyDefaultThemeComponentsRE` regex in src/node/build/bundle.ts.
+export { default as VPButton } from './components/VPButton.vue'
 export { default as VPHomeHero } from './components/VPHomeHero.vue'
 export { default as VPHomeFeatures } from './components/VPHomeFeatures.vue'
 export { default as VPHomeSponsors } from './components/VPHomeSponsors.vue'
 export { default as VPDocAsideSponsors } from './components/VPDocAsideSponsors.vue'
+export { default as VPSponsors } from './components/VPSponsors.vue';
```

Comment:
add `VPSponsors` to `lazyDefaultThemeComponentsRE` as mentioned in the note above